data/random-teams.ts

```diff
@@ -2047,9 +2047,7 @@ export class RandomTeams {
 		// Picks `n` random pokemon--no repeats, even among formes
 		// Also need to either normalize for formes or select formes at random
 		// Unreleased are okay but no CAP
-		const last = [0, 151, 251, 386, 493, 649, 721, 807, 898, 1010][this.gen];
```

Comment:
this is not the way to go about this. just update the `1010` to `1017`

Reply:
Is there any reason for that being there that I'm not seeing? Best I can tell its _only_ effect is causing a bug every time a new mon drops

Reply:
well first, by removing this, you're removing the generation check, so this wont respect generations anymore